awx/ui_next/src/components/ContentError/NotFoundError.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { t } from '@lingui/macro';
+import styled from 'styled-components';
+import { withI18n } from '@lingui/react';
+import {
+  Title,
+  EmptyState as PFEmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+} from '@patternfly/react-core';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons';
+import ErrorDetail from '@components/ErrorDetail';
+
+const EmptyState = styled(PFEmptyState)`
+  width: var(--pf-c-empty-state--m-lg--MaxWidth);
+`;
+
+function NotFoundError({ i18n, error, children }) {
+  return (
+    <EmptyState>
```

Comment:
I'm concerned about growing a new content error for each kind of http error.

Can we keep our treatment of api errors generic by surfacing the details provided by the API for the message, inside of `ContentError` instead?



![Screenshot from 2019-08-08 11-51-49](https://user-images.githubusercontent.com/9753817/62718061-21557f00-b9d3-11e9-9c54-00f74683da0d.png)

As you can see, the api-provided message isn't as informative right now, but I'm wondering if we can change it api-side.